src/js/components/inspectors/MultiValueProperty.tsx

```diff
@@ -0,0 +1,178 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
+import { resetMarkVisual } from '../../actions/markActions';
+import { PrimType } from '../../constants/primTypes';
+import { State } from '../../store';
+import { DraggingStateRecord } from '../../store/factory/Inspector';
+import { Property } from './Property';
+
+interface OwnState {
+  dragOver?: number;
+}
+
+// hunch: probably actually needs to be a bunch of different types anded to a base type
+// instead of one type with everything optional
+// but it would take a long time to figure out what the mutually exclusive fields are
+interface OwnProps {
```

Comment:
this props interface is a mess because `Property.tsx` doesn't have a consistent interface across all its uses. Since we're writing this new component, and can control where it's used, let's clean up the props interface to remove optionals and remove props that aren't used here.